packages/usa-tooltip/src/index.js

```diff
@@ -371,16 +377,15 @@ const tooltip = behavior(
         }
       },
       [TOOLTIP_TRIGGER](e) {
-        const { trigger, body } = getTooltipElements(e.target);
+        const { trigger, body, wrapper } = getTooltipElements(e.target);
 
         showToolTip(body, trigger, trigger.dataset.position);
+        wrapper.addEventListener("mouseleave", closeTooltip);
```

Comment:
I was not able to get `mouseleave` to work using the standard behavior method. so instead, I triggered the event listener only when the tooltip is active. Please let me know if you see a better way.

Reply:
We have to use `mouseout` for it to work with behaviors.

Not opposed to `mouseleave`, but we should add it once on `init()` and remove it on teardown.

---

Wrapper and the `TOOLTIP` (line 370) share the same class, so you _could_ add the show/hide behavior in there instead of relying on trigger.